### PR TITLE
Recipe list

### PR DESCRIPTION
**Pull request by Valerii2022**

### CODE REVIEW

File: src/components/PopularRecipes/PopularRecipes.jsx [hunk 1]
```diff
@@ -1,10 +1,257 @@
...
+      avatar:
+        'https://st3.depositphotos.com/9998432/13335/v/450/depositphotos_133352156-stock-illustration-default-placeholder-profile-icon.jpg',
+    },
+    description:
+      'A Chinese-American classic made with tender pieces of pork, bell peppers, and pineapple in a sweet and tangy sauce. Served with rice.',
+    thumb:
+      'https://res.cloudinary.com/dhap6yrcb/image/upload/f_auto/v1717776201/recipes-init/qmxepodwqbzg5iocmca4',
+  },
+];
+// ==============================
```
Comment (Iryna-Holova) [inline src/components/PopularRecipes/PopularRecipes.jsx:248]: move data to src/temp folder

File: src/components/PopularRecipes/PopularRecipes.module.css [hunk 2]
```diff
@@ -0,0 +1,15 @@
+.title {
+    font-weight: 800;
+    font-size: 18px;
+    line-height: 1.33;
+    letter-spacing: -0.02em;
+    text-transform: uppercase;
+    color: var(--color-black);
```
Comment (Iryna-Holova) [inline src/components/PopularRecipes/PopularRecipes.module.css:7]: --color-main

File: src/components/RecipeList/RecipeCard/RecipeCard.jsx [hunk 3]
```diff
@@ -1,5 +1,83 @@
...
+      return text.slice(0, 77) + '...';
+    }
+    if (tablet && text.length >= 85) {
+      return text.slice(0, 85) + '...';
+    }
+    if (mobile && text.length >= 100) {
+      return text.slice(0, 100) + '...';
+    }
+    return text;
+  };
```
Comment (Iryna-Holova) [inline src/components/RecipeList/RecipeCard/RecipeCard.jsx:26]: truncate in css

File: src/components/RecipeList/RecipeCard/RecipeCard.module.css [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+    flex-grow: 1;
+}
+
+.title {
+    font-weight: 800;
+    font-size: 18px;
+    line-height: 1.33;
+    letter-spacing: -0.02em;
+    text-transform: uppercase;
+    color: var(--color-black);
```
Comment (Iryna-Holova) [inline src/components/RecipeList/RecipeCard/RecipeCard.module.css:30]: --color-main

File: src/components/RecipeList/RecipeCard/RecipeCard.module.css [hunk 5]
```diff
@@ -0,0 +1,137 @@
...
+
+.owner_btn {
+    display: flex;
+    gap: 8px;
+    align-items: center;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 1.43;
+    letter-spacing: -0.02em;
+    color: var(--color-black-90);
```
Comment (Iryna-Holova) [inline src/components/RecipeList/RecipeCard/RecipeCard.module.css:54]: --color-main-90

File: src/components/RecipeList/RecipeCard/RecipeCard.module.css [hunk 6]
```diff
@@ -0,0 +1,137 @@
...
+    align-items: center;
+}
+
+.icon {
+    width: 16px;
+    height: 16px;
+}
+
+.favorite_heart {
+    background-color: var(--color-black);
```
Comment (Iryna-Holova) [inline src/components/RecipeList/RecipeCard/RecipeCard.module.css:86]: --color-main

File: src/components/RecipeList/RecipeCard/RecipeCard.module.css [hunk 7]
```diff
@@ -0,0 +1,137 @@
...
+    color: var(--color-white);
```
Comment (Iryna-Holova) [inline src/components/RecipeList/RecipeCard/RecipeCard.module.css:87]: --color-secondary

**Review by Iryna-Holova — COMMENTED** [review]
fix please

File: src/components/PopularRecipes/PopularRecipes.jsx (hunk 1 above)
Comment (Valerii2022) [reply src/components/PopularRecipes/PopularRecipes.jsx:248]: ok

File: src/components/PopularRecipes/PopularRecipes.module.css (hunk 2 above)
Comment (Valerii2022) [reply src/components/PopularRecipes/PopularRecipes.module.css:7]: ok

File: src/components/RecipeList/RecipeCard/RecipeCard.module.css (hunk 4 above)
Comment (Valerii2022) [reply src/components/RecipeList/RecipeCard/RecipeCard.module.css:30]: ok

File: src/components/RecipeList/RecipeCard/RecipeCard.module.css (hunk 5 above)
Comment (Valerii2022) [reply src/components/RecipeList/RecipeCard/RecipeCard.module.css:54]: ok

File: src/components/RecipeList/RecipeCard/RecipeCard.module.css (hunk 6 above)
Comment (Valerii2022) [reply src/components/RecipeList/RecipeCard/RecipeCard.module.css:86]: ok

File: src/components/RecipeList/RecipeCard/RecipeCard.module.css (hunk 7 above)
Comment (Valerii2022) [reply src/components/RecipeList/RecipeCard/RecipeCard.module.css:87]: ok

File: src/components/RecipeList/RecipeCard/RecipeCard.jsx (hunk 3 above)
Comment (Valerii2022) [reply src/components/RecipeList/RecipeCard/RecipeCard.jsx]: ok

**Review by Iryna-Holova — APPROVED** [review]

**Pull request merged**